lib/RGBot.js

```diff
@@ -30,18 +33,103 @@ const { Vec3 } = require('vec3');
  *  <b><u>Item</u></b><br>
  *    An Item represents any Entity that can be collected in the player's inventory or hands.
  *    These can be things like weapons and armor that the player equips, crafting materials, or items that can be placed to create a Block.
- *    This last example brings up an important distinction to keep in mind while creating your Bot: an object is an Item when in the Bot's inventory or hand, or when it has been tossed on the ground, but it is a Block once it is placed in the world.
+ *    This last example brings up an important distinction to keep in mind while creating your Bot: an object is an Item when in the bot inventory or hand, or when it has been tossed on the ground, but it is a Block once it is placed in the world.
  *
  *  <b><u>Name versus Display Name</u></b><br>
  *    An Entity's name is a unique identifier, and its display name is typically the same or similar identifier but in a human-readable format.
  *    As an example, the Ender Dragon is the readable name, or display name, of the Entity named ender_dragon. Likewise, Grass Block is the display name of the block named grass_block.
- *    This library attempts to accept both the name and display name interchangeably wherever possible, so the identifier you use is up to your own personal tastes.
+ *    This library provides functions to accept the name exclusively, or the name and display name interchangeably, so the identifier you use is up to your own personal tastes.
```

Comment:
I notice that a few of the functions in this class are called `experimental`. Can we add a comment that provides some info on what that means to the user. I am guessing it means that the interface may change?

Reply:
@abeizer , what was your original intent of that tag ?

Reply:
Aaron asked me to add it for things that hadn't been tested thoroughly, as an indication to users that it may be buggy or not work as expected.

Reply:
Thank you for jogging my memory, yeah let's write that down